Configurations/unix-Makefile.tmpl

```diff
@@ -551,7 +551,7 @@ build_docs: build_man_docs build_html_docs ## Create documentation
 build_man_docs: $(MANDOCS1) $(MANDOCS3) $(MANDOCS5) $(MANDOCS7) ## Create manpages
 build_html_docs: $(HTMLDOCS1) $(HTMLDOCS3) $(HTMLDOCS5) $(HTMLDOCS7) ## Create HTML documentation
 
-build_generated: $(GENERATED_MANDATORY)
+build_generated: $(GENERATED_MANDATORY) generate_crypto_objects generate_crypto_bn generate_crypto_conf generate_crypto_asn1
```

Comment:
This is absolutely not enough.  If we do this in build time, please also modify `generate_crypto_objects`, `generate_crypto_bn`, `generate_crypto_conf` and `generate_crypto_asn1` to write into `$(BLDDIR)` rather than `$(SRDDIR)`.

Also, if we *really* wanna do this properly, they should probably appear in the top `build.info` as `GENERATE` lines and be included in `DEPEND[]`, which also means that the scripts that generate them will need a bit of tweaking to interact properly with the invokations generated from `GENERATE` lines.

With all *those* changes, we have a chance to have something that works uniformly on all platforms.

(for `build.info`, you wanna read `doc/internal/man7/build.info.pod`)